src/docs/cookbook/networking/background-parsing.md

```diff
@@ -25,10 +25,10 @@ On Android, this means scheduling work on a different thread.
 In Flutter, you can use a separate [Isolate][].
 This recipe uses the following steps:
 
-  1. Add the `http` package.
-  2. Make a network request using the `http` package.
-  3. Convert the response into a list of photos.
-  4. Move this work to a separate isolate.
+  1. Add the `http` package
+  2. Make a network request using the `http` package
+  3. Convert the response into a list of photos
+  4. Move this work to a separate isolate
```

Comment:
Shouldn't we be using punctuation here? 

https://developers.google.com/style/lists

![Screen Shot 2020-03-10 at 4 30 44 PM](https://user-images.githubusercontent.com/5550175/76368214-ae0b4700-62ec-11ea-8a9d-67368bcb1c33.png)

Reply:
![Screen Shot 2020-03-10 at 4 52 51 PM](https://user-images.githubusercontent.com/5550175/76369244-2fb0a400-62f0-11ea-863c-af3d2909d676.png)